services/venice-router/src/main/java/com/linkedin/venice/router/stats/RouteHttpRequestStats.java

```diff
@@ -57,25 +57,24 @@ static class InternalHostStats extends AbstractVeniceStats {
     private final Sensor unhealthyPendingQueueDuration;
     private final Sensor unhealthyPendingRateSensor;
     private AtomicLong pendingRequestCount;
+    private final Sensor pendingRequestCountSensor;
 
     public InternalHostStats(MetricsRepository metricsRepository, String hostName) {
       super(metricsRepository, StatsUtils.convertHostnameToMetricName(hostName));
-      pendingRequestCount = new AtomicLong();
-      // pendingRequestCountSensor =
-      // registerSensor(new AsyncGauge((ignored, ignored2) -> pendingRequestCount.get(), "pending_request_count"));
+      this.pendingRequestCount = new AtomicLong();
+      this.pendingRequestCountSensor = registerSensor("pending_request_count", new Max());
 
-      unhealthyPendingQueueDuration = registerSensor(
+      this.unhealthyPendingQueueDuration = registerSensor(
```

Comment:
while we are at it, can we reduce the stats for this sensor ? just avg or max should be enough?

Reply:
or even remote it altogether, do not recall using this metric in a long time

Reply:
This metric can be useful when some nodes are marked as unhealthy, right?